release-notes/opensearch-dashboards-notifications.release-notes-3.0.0.0-alpha1.md

```diff
@@ -0,0 +1,6 @@
+## Version 3.0.0.0-alpha1 Release Notes
+
+Compatible with OpenSearch 3.0.0.0-alpha1
+
+### Maintenance
+* [Release 3.0] Add alpha1 qualifier. ([#333](https://github.com/opensearch-project/dashboards-notifications/pull/333))
```

Comment:
334?

Reply:
I saw other plugins didn't list the release notes PR in the notes, so I followed that pattern for ours.

Reply:
Discussed offline. No need to change this reference.